### PR TITLE
fix: fix accessibility issues in register embedded experience

### PR DESCRIPTION
**Pull request by syedsajjadkazmii**
VAN-1541, VAN-1551, VAN-1543

### Description

- Fix honor code and TOS styling
- Fix padding issue

#### JIRA

[VAN-1541](https://2u-internal.atlassian.net/browse/VAN-1541)
[VAN-1551](https://2u-internal.atlassian.net/browse/VAN-1551)
[VAN-1543](https://2u-internal.atlassian.net/browse/VAN-1543)

#### How Has This Been Tested?

locally

#### Screenshots/sandbox (optional):

Screenshot 2023-07-24 at 3 37 18 PM
Screenshot 2023-07-24 at 3 37 39 PM
Screenshot 2023-07-24 at 3 37 48 PM

#### Merge Checklist

* [ ] If your update includes visual changes, have they been reviewed by a designer? Send them a link to the Sandbox, if applicable.
* [ ] Is there adequate test coverage for your changes?

#### Post-merge Checklist

* [ ] Deploy the changes to prod after verifying on stage or ask **@openedx/vanguards** to do it. 
* [ ] 🎉 🙌 Celebrate! Thanks for your contribution.

### CODE REVIEW

File: src/register/registrationFields/HonorCode.jsx [hunk 1]
```diff
@@ -30,12 +30,22 @@ const HonorCode = (props) => {
           values={{
             platformName: getConfig().SITE_NAME,
             tosAndHonorCode: (
-              <Hyperlink variant="muted" destination={getConfig().TOS_AND_HONOR_CODE || '#'} target="_blank">
+              <Hyperlink
```
Comment (shahbaz-arbisoft) [inline src/register/registrationFields/HonorCode.jsx:33]: Isn't the `variant` attribute not needed now?

File: src/register/registrationFields/HonorCode.jsx [hunk 2]
```diff
@@ -30,12 +30,22 @@ const HonorCode = (props) => {
...
+                className="inline-link"
+                destination={getConfig().TOS_AND_HONOR_CODE || '#'}
+                target="_blank"
+                showLaunchIcon={false}
+              >
                 {formatMessage(messages['terms.of.service.and.honor.code'])}
               </Hyperlink>
             ),
             privacyPolicy: (
-              <Hyperlink variant="muted" destination={getConfig().PRIVACY_POLICY || '#'} target="_blank">
```
Comment (shahbaz-arbisoft) [inline src/register/registrationFields/HonorCode.jsx:38]: Isn't the `variant` attribute not needed now?

**Review by shahbaz-arbisoft — APPROVED** [review]
Was this [ticket](https://2u-internal.atlassian.net/browse/VAN-1551) issue resolved after adding padding to the form?

File: src/register/registrationFields/HonorCode.jsx (hunk 1 above)
Comment (syedsajjadkazmii) [reply src/register/registrationFields/HonorCode.jsx:33]: nope, we want to make it visible and different from normal text

File: src/register/registrationFields/HonorCode.jsx (hunk 2 above)
Comment (syedsajjadkazmii) [reply src/register/registrationFields/HonorCode.jsx:38]: answered above.

**Comment by syedsajjadkazmii** [comment]
> Was this [ticket](https://2u-internal.atlassian.net/browse/VAN-1551) issue resolved after adding padding to the form?

yes

**Pull request merged**